packages/block-editor/src/components/inspector-controls-tabs/index.js

```diff
@@ -49,8 +49,7 @@ export default function InspectorControlsTabs( {
 							tabId={ tab.name }
 							render={
 								<Button
-									// TODO: Switch to `true` (40px size) if possible
-									__next40pxDefaultSize={ false }
+									__next40pxDefaultSize
```

Comment:
**Before:**
![before](https://github.com/user-attachments/assets/9264b44a-ca11-48d7-82e7-4670f5ee8c28)

**After:**
![after](https://github.com/user-attachments/assets/439e8c0e-8cd8-40bb-b3c8-73cbd776f476)

`__next40pxDefaultSize` has no effect here because the parent `div` has default `height: 48px`

Reply:
It would be tricky to get rid of all the overrides here. I wonder if it even makes sense to use a `Button` component for such a heavily customized use case.

Reply:
Agreed. Could we try to switch to a regular `button` ? Since `Tabs.Tab` already renders a `button` by default, we could avoid using the `render` prop altogether.

Reply:
Applying this diff should allow us to refactor away from `Button` here

```diff
diff --git a/packages/block-editor/src/components/block-inspector/style.scss b/packages/block-editor/src/components/block-inspector/style.scss
index bdbf3660d9..8aecd85a2c 100644
--- a/packages/block-editor/src/components/block-inspector/style.scss
+++ b/packages/block-editor/src/components/block-inspector/style.scss
@@ -49,4 +49,7 @@
 
 .block-editor-block-inspector__tab-item {
 	flex: 1 1 0px;
+
+	display: flex;
+	justify-content: center;
 }
diff --git a/packages/block-editor/src/components/inspector-controls-tabs/index.js b/packages/block-editor/src/components/inspector-controls-tabs/index.js
index 35917d9bd7..c8d9536aeb 100644
--- a/packages/block-editor/src/components/inspector-controls-tabs/index.js
+++ b/packages/block-editor/src/components/inspector-controls-tabs/index.js
@@ -2,7 +2,8 @@
  * WordPress dependencies
  */
 import {
-	Button,
+	Icon,
+	Tooltip,
 	privateApis as componentsPrivateApis,
 } from '@wordpress/components';
 import { store as preferencesStore } from '@wordpress/preferences';
@@ -43,26 +44,27 @@ export default function InspectorControlsTabs( {
 		<div className="block-editor-block-inspector__tabs">
 			<Tabs defaultTabId={ initialTabName } key={ clientId }>
 				<Tabs.TabList>
-					{ tabs.map( ( tab ) => (
-						<Tabs.Tab
-							key={ tab.name }
-							tabId={ tab.name }
-							render={
-								<Button
-									__next40pxDefaultSize
-									icon={
-										! showIconLabels ? tab.icon : undefined
-									}
-									label={
-										! showIconLabels ? tab.title : undefined
-									}
+					{ tabs.map( ( tab ) =>
+						showIconLabels ? (
+							<Tabs.Tab
+								key={ tab.name }
+								tabId={ tab.name }
+								className={ tab.className }
+							>
+								{ tab.title }
+							</Tabs.Tab>
+						) : (
+							<Tooltip text={ tab.title } key={ tab.name }>
+								<Tabs.Tab
+									tabId={ tab.name }
 									className={ tab.className }
+									aria-label={ tab.title }
 								>
-									{ showIconLabels && tab.title }
-								</Button>
-							}
-						/>
-					) ) }
+									<Icon icon={ tab.icon } />
+								</Tabs.Tab>
+							</Tooltip>
+						)
+					) }
 				</Tabs.TabList>
 				<Tabs.TabPanel tabId={ TAB_SETTINGS.name } focusable={ false }>
 					<SettingsTab showAdvancedControls={ !! blockName } />

```